### PR TITLE
Alerts cleanup

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Formatting, grammar, and IA cleanup

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/alerts/monitors/alert-grouping.md [hunk 1]
```diff
@@ -115,7 +116,7 @@ A user wants to get an alert if all hosts from a given service has stopped sendi
 
 This alert can be useful if you suspect that one of your collectors has stopped sending data. Once you set this up, you'll get an alert about collector if it's stopped sending data. This alert will work without any issues, even if you add new collectors to your Sumo Logic account.
 
-* **Query**:  
+**Query**:
```
Comment (kimsauce) [inline docs/alerts/monitors/alert-grouping.md:119]:
```suggestion
* **Query**:  
```

File: docs/alerts/monitors/alert-grouping.md [hunk 2]
```diff
@@ -68,7 +69,7 @@ A user wants to create a monitor to track CPU and be notified if any node within
 * **Query**: `metric=CPU_sys`
 * **Group Condition**: Service, env <br/><img src={useBaseUrl('img/monitors/usecase2.png')} alt="alert-grouping" />
 * **Alert Evaluation Logic**: If `CPU_sys` for any node within a service,env is greater than `60`, then an alert notification will be generated for that service within a given environment (if it was not already generated).
-* **Recovery Evaluation Logic:**
+* **Recovery Evaluation Logic**:
```
Comment (kimsauce) [inline docs/alerts/monitors/alert-grouping.md:72]:
```suggestion
* **Recovery Evaluation Logic:**
```

File: docs/alerts/monitors/alert-grouping.md [hunk 3]
```diff
@@ -106,7 +107,7 @@ A user wants to get an alert if all hosts from a given service has stopped sendi
 * **Query**: `metric=CPU_sys`
 * **Group Condition**: service <br/><img src={useBaseUrl('img/monitors/usecase5.png')} alt="alert-grouping" />
 * **Alert Evaluation Logic**: If all the hosts stop sending data (`CPU_sys` metric is not being sent) then generate an alert for a given service, then an alert notification will be generated for that service (if it was not already generated). The list of hosts for a service will be computed and updated on a periodic basis.
-* **Recovery Evaluation Logic:**
+* **Recovery Evaluation Logic**:
```
Comment (kimsauce) [inline docs/alerts/monitors/alert-grouping.md:110]:
```suggestion
* **Recovery Evaluation Logic:**
```

**Review by tarunk2 — APPROVED** [review]
LGTM, thanks!

**Review by MariePMcGarry — APPROVED** [review]

**Pull request merged**